### PR TITLE
fixes #33: Only listen to functionTarget.

### LINKED ISSUE #33: functions called from browser execute twice
Although curl will only call the function once.
Chrome load or refresh calls the function twice.
A bit confusing when debugging something more complex than HelloWorld.

Very easy to see:
```
exports.helloWorld = (req, res) => {
    console.log('running HelloWorld'); // <-- oh no not again
    res.send('Hello, World');
  };
```

Is there a way to stop this?

### PR DESCRIPTION
**Pull request by grant**
- Specifies _exact_ function target instead of `*`.
- Fixes #33.
- Ran linter.

### CODE REVIEW

**Comment by hdp617** [comment]
Could you also update the test?

**Comment by grant** [comment]
> Could you also update the test?
@hdp617 Addressed comment and fixed Travis CI tests.

PTAL.

File: test/invoker.ts [hunk 1]
```diff
@@ -34,7 +34,8 @@ describe('request to HTTP function', () => {
       (req: express.Request, res: express.Response) => {
         res.send(req.body.text.toUpperCase());
       },
-      invoker.SignatureType.HTTP
+      invoker.SignatureType.HTTP,
+      '/'
```
Comment (hdp617) [inline test/invoker.ts]: This is an odd function target. I suggest something like 'testFunction'.

File: test/invoker.ts [hunk 2]
```diff
@@ -34,7 +34,8 @@ describe('request to HTTP function', () => {
...
     );
     return supertest(server)
       .post('/')
```
Comment (hdp617) [inline test/invoker.ts]: `.post('/'testFunction')`?

File: test/invoker.ts (hunk 1 above)
Comment (grant) [reply test/invoker.ts]: Applied suggestion of `testFunction`.

File: test/invoker.ts (hunk 2 above)
Comment (grant) [reply test/invoker.ts]: Done.

**Pull request merged**